Correct document error in traffic-control.md

hongliangl opened the pull request.

hongliangl commented:
/skip-all

tnqn reviewed (approved):
LGTM

hongliangl commented:
> @hongliangl commit not signed

Done

antoninbas commented:
@hongliangl you haven't pushed

hongliangl commented:
> @hongliangl you haven't pushed

Done. It seems that the approval won't be dismissed when only the commit message is modified.

antoninbas commented:
Looks like the DCO check is still failing though, can you double check?

tnqn commented:
```
Expected "Hongliang Liu [75655411+hongliangl@users.noreply.github.com](mailto:75655411+hongliangl@users.noreply.github.com)", but got "Hongliang Liu [lhongliang@vmware.com](mailto:lhongliang@vmware.com)".
```
@hongliangl did you edit the commit on github directly?

hongliangl commented:
> @hongliangl you haven't pushed

I have added the sign information in github codespace, and I could see the sign in commit message, but 

> ```
> Expected "Hongliang Liu [75655411+hongliangl@users.noreply.github.com](mailto:75655411+hongliangl@users.noreply.github.com)", but got "Hongliang Liu [lhongliang@vmware.com](mailto:lhongliang@vmware.com)".
> ```
> 
> @hongliangl did you edit the commit on github directly?

I modified that in github codespace directly, them force push it.

hongliangl commented:
> DCO passed. I have tried the following command in codespace then force push:
> 
> ```
> git rebase HEAD~1 --signoff
> ```

Resolved this by modify the email address of git config in github codespace, not the above.

tnqn commented:
/skip-all
